src/test/java/com/redhat/cloud/notifications/templates/TestPoliciesTemplate.java

```diff
@@ -185,11 +166,10 @@ public void testDailyEmailBodyOnePoliciesAndOneSystem() {
         payload.put("unique_system_count", 1);
 
         String result = Policies.Templates.dailyEmailBody()
-                .data("payload", payload)
+                .data("action", Map.of("context", payload))
                 .render();
 
         assertTrue(result.contains("<b>1 policy</b> triggered on <b>1 system</b>"));
-        assertFalse(result.contains("NOT_FOUND"), "A replacement was not correctly done");
```

Comment:
This was removed because of the `quarkus.qute.property-not-found-strategy`, right?

Reply:
Yes, I think is better to hard fail so is easier for devs working on the templates to know right away that something is wrong.